service/backend/pchain/indexer/testdata/outs/1000004.json

```diff
@@ -2,12 +2,12 @@
   "id": "2Kbis7MxM1MUqdfNxKdhP7FAhEdNYBHDvPivFWmGktDV9Au5nL",
   "type": "*blocks.ApricotCommitBlock",
   "parent": "DGRnAy4HYV52uWCRKZV6UXw9s9qZKCat5Xss8PANDaR5mngUz",
-  "timestamp": 1599696000000,
+  "timestamp": 1638504099,
   "height": 1000005,
   "transactions": [],
   "proposer": {
-    "id": "11111111111111111111111111111111LpoYY",
```

Comment:
Not sure why we have primary network ID here instead of proposer block ID and proposer block parent

Reply:
Probably it's because it is an uninitialized ids.ID, just happens to be the same as P-chain id

Reply:
yes indeed. cb58 representation of empty ID is the one here